server/public/model/websocket_message_test.go

```diff
@@ -41,7 +41,7 @@ func TestWebSocketEventImmutable(t *testing.T) {
 		require.Fail(t, "pointers should not be the same")
 	}
 	require.NotEqual(t, m.EventType(), newM.EventType())
-	require.Equal(t, newM.EventType(), "new_event")
+	require.Equal(t, string(newM.EventType()), "new_event")
```

Comment:
While I'm not against using type casting, I wonder if instead a "real" event type should be used here.

Reply:
good point, [switched](https://github.com/mattermost/mattermost/pull/25195/commits/038372455a49c0d7f5754892e6cb7b198991da2e) to use real event names